src/common/HTTPRequest.ts

```diff
@@ -295,12 +433,29 @@ export class HTTPRequest {
    * ```
    *
    * @param overrides - optional overrides to apply to the request.
+   * @param priority - (optional, default 0). Priority 0 will execute immediately.
+   * Priorities > 0 will be resolved using coopeative handling rules.
    */
-  async continue(overrides: ContinueRequestOverrides = {}): Promise<void> {
+  async continue(
+    overrides: ContinueRequestOverrides = {},
+    priority = LEGACYY_DEFAULT_INTERCEPT_PRIORITY
```

Comment:
WDYT about typing this as `priority?: number` and checking for `undefined` instead of `LEGACYY_DEFAULT_INTERCEPT_PRIORITY`

Reply:
I like this idea.